composer.json

```diff
@@ -43,32 +43,32 @@
         "symfony/web-profiler-bundle": "^5.3",
         "symfony/webpack-encore-bundle": "^1.8",
         "symfony/yaml": "^5.3",
-        "symplify/amnesia": "9.4.22",
-        "symplify/autowire-array-parameter": "9.4.22",
-        "symplify/composer-json-manipulator": "9.4.22",
-        "symplify/easy-testing": "9.4.22",
-        "symplify/markdown-diff": "9.4.22",
-        "symplify/package-builder": "9.4.22",
-        "symplify/php-config-printer": "9.4.22",
-        "symplify/rule-doc-generator": "9.4.22",
-        "symplify/simple-php-doc-parser": "9.4.22",
-        "symplify/skipper": "9.4.22",
-        "symplify/smart-file-system": "9.4.22",
-        "symplify/symfony-php-config": "9.4.22",
-        "symplify/symplify-kernel": "9.4.22"
+        "symplify/amnesia": "^9.4.23",
```

Comment:
@TomasVotruba do you think it should be `9.4.23` or `^9.4.23` ?

Reply:
Definitely `^9.4`, I was probably testing something and forgot to open the dependency up